scripts/xnat/miqa_file_generation.py

```diff
@@ -75,19 +81,27 @@ def convert_dataframe_to_new_format(df, image_extensions, verbose=False):
         frame_locations.sort(key=lambda path: path.name)
         for index, frame_location in enumerate(frame_locations):
             frame_number = row["scan_id"] if len(frame_locations) < 2 else index
+            experiment = row["xnat_experiment_id"]
+            subject_id = ""
+            session_id = experiment  # TODO populate correctly
+            scan_link = ""
+            if session:
+                scan_link = session.get_xnat_session_address(experiment)
+            if subject_mapping and experiment in subject_mapping:
+                subject_id = subject_mapping[experiment]
+
             new_rows.append(
                 [
                     project_name,  # project_name
-                    row["xnat_experiment_id"],  # experiment_name
-                    f"{row['xnat_experiment_id']}_{row['scan_type']}",  # scan_name
+                    experiment,  # experiment_name
+                    f"{experiment}_{row['scan_type']}",  # scan_name
```

Comment:
You could set scan_type = row['scan_type'] above to be consistent with what you did with experiment

Reply:
Changed in [61094ae](https://github.com/sibis-platform/ncanda-data-integration/pull/498/commits/61094aecabf3f47c93c9b811f9a4bdec892e0cae)